### PR TITLE
Implementing Mat.FromImage helper function

### PR DESCRIPTION
**Pull request by denismakogon**
Was: #156

### CODE REVIEW

File: core.go [hunk 1]
```diff
@@ -486,35 +486,33 @@ func (m *Mat) ToImage() (image.Image, error) {
 }
 
 // FromImage converts go Image to the Mat object
-func (m *Mat) FromImage(img image.Image, imgType ImageMagicType, mt MatType) error {
+func FromImage(img image.Image, imgType ImageMagicType, mt MatType) (*Mat, error) {
```
Comment (deadprogram) [inline core.go]: Seems like should probably just return a `Mat` not a `*Mat`.

**Comment by deadprogram** [comment]
Hi @denismakogon looks like the tests are failing now on both Linux and Windows, just in case you had not noticed.

**Comment by denismakogon** [comment]
Hi! @deadprogram, I've seen that didn't have time to investigate those, so, probably would need certain help with that if possible.

**Comment by deadprogram** [comment]
Hi @denismakogon now on top of the failing tests there is also a merge conflict for this PR. Seems like it will need some more love in order to land...

**Comment by deadprogram** [comment]
Seems like the approach in #228 is more efficient @denismakogon what do you think?

**Comment by denismakogon** [comment]
looks good, will try updating PR ASAP.

**Comment by edwvee** [comment]
Have looked code through. Tried common thing:
```go
//img is finely decoded jpeg
bounds := img.Bounds()
	x := bounds.Dx()
	y := bounds.Dy()
	data := []byte{}
	w := bytes.NewBuffer(data)
	jpeg.Encode(w, img, &jpeg.Options{Quality: 90})
	data = w.Bytes()
	mat, err := gocv.NewMatFromBytes(y, x, gocv.MatTypeCV8UC3, data)

	if err != nil {
		log.Fatal(err)
	}
```
No error. But when i launched ```mat.ToImage()```  - received panic. Who or what is wrong: the way I uderstood this PR's code, this PR or mat.ToImage()?

**Comment by deadprogram** [comment]
This PR has now been superseded by #234 which has been merged into the `dev` branch. Now closing.

**Pull request closed without merging**